components/ModDashboardLeftNavbar.vue

```diff
@@ -79,6 +78,9 @@ function createCountsForSubmissionListView(submissions: Submission[]) {
         if (submission.isRejected) {
             statusCountObject.rejectedCount += 1
         }
+        if (!submission.isRejected && !submission.isApproved && !submission.isUnderReview) {
```

Comment:
What do you think about saving these to a variable and just matching the condition on the variable here?

Reply:
Nice call @theyokohamalife i see you picking up good habits 🙌🙌